frontend/.eslintrc.cjs

```diff
@@ -22,6 +22,8 @@ module.exports = {
         'react/jsx-uses-react': 'off',
         'import/no-unresolved': 'off',
         'import/no-absolute-path': 'off',
+        'import/no-undef': 'off',
+        "object-curly-newline": "off",
```

Comment:
🔍

Reply:
Lol thanks for catching that double quote.